zookeeper-server/src/test/java/org/apache/zookeeper/server/quorum/QuorumPeerMainMultiAddressTest.java

```diff
@@ -230,6 +233,29 @@ public void shouldReconfigIncrementally_IPv6() throws Exception {
     checkIfZooKeeperQuorumWorks(newQuorumConfig);
   }
 
+  @Test(expected = KeeperException.class)
```

Comment:
this way of testing for an expected exception is too broad, you don't know where KeeperException has been thrown.
I suggest to push the assertion around the  ReconfigTest.reconfig call

Reply:
thanks, it's nicer that way indeed, I will fix it